src/doc.js

```diff
@@ -127,49 +127,84 @@ var prototype = node.derive({
                     if ( !word.text.parts || !word.text.parts.length || !word.text.parts[0] ) {
                         continue;
                     }
-                    var text = word.text.parts[0];
-                    if ( text.run.underline === true ) {
-                        underLines.push({
-                            baseline: line.baseline,
-                            width: word.width, // text width + space width,
-                            left,
-                            color: text.run.color || this.defaultFormatting.color,
-                        })
-                    }
 
-                    if ( text.run.strikeout === true ) {
-                        strikLines.push({
-                            ascent: word.ascent,
+                    for (let k = 0; k < word.text.parts.length; k++) {
```

Comment:
This change was done to fix a bug, when a word is partially formatted considering `word.text.parts[0]` is wrong, all the parts should be considered  i.e. `for (let k = 0; k < word.text.parts.length; k++) {`